### DISCUSSION TITLE
Support for taking ownership of existing containers

### DISCUSSION THREAD

**Discussion by 00o-sh** [louislam/dockge#155] [Feature Request]
This can be done by generating a docker compose example project that does this

[docker-autocompose](https://github.com/Red5d/docker-autocompose)

I'm on my phone not sure how to hyperlink that link

**Comment by Zaid-maker** [comment]
`[]()` that's how we do!

**Reply by 00o-sh** [reply]
Seems to fall away when I do that

**Comment by Jayknightfr** [comment]
Would be great to have this feature implemented. I've used autocompose to recreate all the docker-compose files i lost when reinstalling portainer. Works great! I assumed Dockge was doing the same thing. Great tool nonetheless, we just need to use autocompose manually in the meantime.
